client/src/pages/AnnouncementPage.js

```diff
@@ -0,0 +1,84 @@
+import React from "react";
+import Sidebar from "../components/Sidebar";
+import SelectDropDownComponent from "../components/Dropdown";
+
+const AnnouncementPage = () => {
+  return (
+    <div className="flex w-full">
+      <Sidebar />
+      <div className="container w-full">
+        <h1 className="font-bold p-3 text-lg">Announcements</h1>
+        <hr></hr>
+        <div class="block p-6 rounded-lg shadow-lg bg-white w-full">
```

Comment:
- [ ] please make this as a component - `AnnouncementCard` so that it's reusable.